absl/debugging/failure_signal_handler.cc

```diff
@@ -54,7 +54,7 @@
 #include "absl/debugging/internal/examine_stack.h"
 #include "absl/debugging/stacktrace.h"
 
-#ifndef _WIN32
+#if !defined(_WIN32) && !defined(__wasi__)
```

Comment:
It seems that Emscripten does support this codepath while wasi doesn't so I added a wasi-specific option here